config.yml.example

```diff
@@ -60,8 +60,10 @@ outgoing_smtp:
   password: "duhnsentqrxgadfh"
 
 #                           ##### site_name #####
-# Set the hostname of the website universally. The hostname set here will
-# reappear multiple times across the project. It is important for this to be an
-# existing DNS record.
+# Creates a resolveable hostname used across the website universally. The site
+# name set here reappears multiple times across the project. It is important
+# for this to be an existing DNS record.
 
-site_name: "localhost"
+site_name:
+  prefix: "http://"
```

Comment:
hardcoding http like this makes me worried.  Specifically mandating the "://" This should be changed from prefix to protocol with two possible options ['http', 'https'] with some help text above.  then the "://" should be added manually in the backend.

Reply:
@nlMeminger Fair feedback. You challenged me to think of something smarter. I think I addressed this in commit e219d5b001f40c67489551690e581815b1ba7f28. Please take a look and re-review.